project1/report.Rmd

```diff
@@ -59,16 +58,126 @@ data[, serum_cat := cut(serum_bilirubin, c(0, 1.1, 3.3, max(serum_bilirubin)))]
 
 # Analysis of the linear regression
 
+```{r, echo=TRUE, eval=TRUE}

```

Comment:
You do not need to include `echo=TRUE` and `eval=TRUE` in each of these blocks, just leave it as `r`. The default setting for any code chunk is `eval=TRUE` so the code will run. As for `echo`, I think it's better to leave that to the default setting specified in line 16, since we will just see the code in the code appendix. So can just leave every code chunk as starting with `{r}`, no other arguments.